boxoffice/mailclient.py

```diff
@@ -55,15 +55,19 @@ def send_line_item_cancellation_mail(line_item_id, subject="Ticket Cancellation"
 
 
 @job('boxoffice')
-def send_ticket_assignment_mail(line_item_id):
+def send_ticket_assignment_mail(line_item_id, past_assignee):
```

Comment:
This method can be litte more lightweight. It can just accept a list of recipients from the caller, instead of having knowledge of a 'past assignee'.